components/analytics/table_chart.tsx

```diff
@@ -1,33 +1,23 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import {OverlayTrigger, Tooltip} from 'react-bootstrap';
 
 import Constants from 'utils/constants';
 
-export default class TableChart extends React.PureComponent {
-    static propTypes = {
-
-        /*
-         * Table title
-         */
-        title: PropTypes.node.isRequired,
-
-        /*
-         * Table data
-         */
-        data: PropTypes.arrayOf(
-            PropTypes.shape({
-                name: PropTypes.string.isRequired,
-                tip: PropTypes.string.isRequired,
-                value: PropTypes.node.isRequired,
-            })
-        ).isRequired,
-    };
+type TableItem = {
+    name: string;
+    tip: string;
+    value: React.ReactNode;
+}
+type TableChartProps = {
```

Comment:
nit: Please change name to `Props` only.
```suggestion
type Props = {
```